apistructs/pipeline_yml.go

```diff
@@ -123,15 +123,21 @@ type SnippetLabel struct {
 	Value string `json:"value"`
 }
 
+type SnippetConfigOrder struct {
```

Comment:
add ut for order.

Reply:
Already, the toString method of SnippetConfig strcut will cover